_posts/0000-01-02-alueib.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![cherryontop-o-cat](https://octodex.github.com/images/cherryontop-o-cat.png)
 {: .center}
 
-Caption here
+Caption here, Caption there, Caption everywhere
```

Comment:
hurray

Reply:
NullPointerException?

Reply:
oh no, it must be your config
cannt reproduce

Reply:
Looks good then!